templates/incl/editor-and-output.html

```diff
@@ -144,6 +144,7 @@
         </div>
         <div id="saveFilesContainer" class="hidden">
           <button id="saveFiles" class="green-btn" onclick="hedyApp.saveMachineFiles();">💾</button>
+          <button id="saveMicrobit" class="green-btn" onclick="hedyApp.saveMachineFiles();">⚡</button>
```

Comment:
We could remove this button but merge the PR, so that we don't run into merge conflicts but also don't bother current users?

Reply:
i'll remove the button now and will push